File: code/e8238.cpp
#include <iostream>
using namespace std;
int main()
{
    int i,n, a[1000],j;
    cin>>n;
    for (i=1; i<=n; i++) cin>>a[i];
    int dem=0;
    for (i=1; i<n; i++)
        for (j=i+1; j<=n; j++)
            if (a[i]==a[j]) dem++;
    cout<<dem;
    return 0;
}

File: code/e8238.pas
var i, n, j, dem: integer;
    a : array[1..1000] of integer;
BEGIN
    readln(n);
    for i:= 1 to n do read(a[i]);
    dem:=0;
    for i:= 1 to n-1 do
        for j:= i+1 to n do
            if a[i]=a[j] then inc(dem);
    write(dem);
END.